### ISSUE TITLE
Add a settings page to customize the name of the HTML file

### ISSUE THREAD

**Issue by bolocholo** [nbeaver/save_link_webextension#2]
This extension is already great, but I'd only add the ability to customize the name of the HTML file. 

Currently, the file name is like: "link-to-Page_title___Site_Name_www.xx.com.link.html". 

The format I'd prefer is: "Page title - Site Name.link.html". 

I personally find it easier to read with spaces instead of underscores, and eliminating the the "link-to" text at the start, and the URL at the end. 

I understand I can hand-edit the filename, but if I had a way to customize to my personal liking it would be great. 

Thanks for your great job.

**Comment by nbeaver** [comment]
I agree that this would be highly useful, but most of the ways I can think of to customize the filename involve free-text input and parsing, which could get very complicated very quickly, so I don't plan on adding this feature until I can find a way that is reasonably simple and robust.

**Comment by siebentod** [comment]
Plus!
Actually it would be useful for me if default name was just the name of the tab, without "link-to-" and "_"-symbols instead of spaces
